pkg/server/sandbox_run.go

```diff
@@ -498,14 +487,36 @@ func (c *criContainerdService) unmountSandboxFiles(rootDir string, config *runti
 	return nil
 }
 
+// setupPod setups up the network for a pod
+func (c *criContainerdService) setupPod(id string, path string, config *runtime.PodSandboxConfig) (string, error) {
+	labels := map[string]string{
```

Comment:
Let's make these string constants.
And let's make the label generation a helper function. The logic is duplicated at least twice.

Reply:
done